tests/trace/test_objects_query_builder.py

```diff
@@ -0,0 +1,134 @@
+import pytest
+
+from weave.trace_server import trace_server_interface as tsi
+from weave.trace_server.objects_query_builder import (
+    ObjectQueryBuilder,
+    _make_conditions_part,
+    _make_limit_part,
+    _make_object_id_conditions_part,
+    _make_offset_part,
+    _make_sort_part,
+)
+
+
+def test_make_limit_part():
```

Comment:
I think a lot of this test suite would be improved if you were to test the public api exposed by your query build as opposed to the private helper functions used internal to construct the final results

Reply:
- unit tests [in this pr]
- end to end tests [exist already]
- object_query_builder public method tests [todo] 

got it

Reply:
will add those

Reply:
the other one is just testing the final constructed sql... i don't think you have any tests that demonstrate the query builders primary function